### PR TITLE
add score characteristic

### PR DESCRIPTION
**Pull request by sk2001git**
added score and allow for edits through edit command else is 0

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Score.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's score in JABPro.
+ */
+public class Score implements Comparable<Score> {
+    public static final String MESSAGE_CONSTRAINTS = "Score should be a non-negative integer";
```
Comment (AriellaCallista) [inline src/main/java/seedu/address/model/person/Score.java:10]: Is there a reason why we're not enforcing a constraint on what the maximum score should be?

File: src/main/java/seedu/address/model/person/Score.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        this.value = value;
+    }
+
+    /**
+     * Returns if a given integer is a valid score.
+     * @param test integer to be tested
+     * @return true if test is a non-negative integer
+     */
+    public static boolean isValidScore(int test) {
+        return test >= 0;
```
Comment (AriellaCallista) [inline src/main/java/seedu/address/model/person/Score.java:30]: As above, perhaps we could enforce a constraint on what the max score is?

**Review by AriellaCallista — APPROVED** [review]
LGTM, left some comments that you might want to address before merging!

File: src/main/java/seedu/address/model/person/Score.java (hunk 1 above)
Comment (sk2001git) [reply src/main/java/seedu/address/model/person/Score.java:10]: Multiple stages of interview may constitute more than a set score. This is the basic structure of score 

Also different companies might want to set a different max score so its not really flexible to include it now

File: src/main/java/seedu/address/model/person/Score.java (hunk 2 above)
Comment (sk2001git) [reply src/main/java/seedu/address/model/person/Score.java:30]: Addressed in the other comment

**Pull request merged**